sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1979,6 +1979,14 @@ object SQLConf {
       .doc("When true, the ArrayExists will follow the three-valued boolean logic.")
       .booleanConf
       .createWithDefault(true)
+
+  val IGNORE_DATA_LOCALITY =
```

Comment:
Could we move this config to [SQLConf.scala#L834](https://github.com/apache/spark/blob/0da290368ae7bf59aa37b3a1aac514808510f484/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala#L834)?

Reply:
Fixed.